spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -64,6 +64,6 @@ public Resource resolve(String location, ResourceLoader resourceLoader) {
 		if (location.startsWith("gs://")) {
 			return new GoogleStorageResource(this.storage, location);
 		}
-		return this.delegate.getResource(location);
+		return null;
```

Comment:
This is obvious breaking change from the `master`  state.
Any particular reason to do this?
Any comments in the commit message reflecting this change like "Why?" and "What?".

One more time Chris Beams' article: https://chris.beams.io/posts/git-commit/